### PR TITLE
Update the token balances REST API to use the real-time balance

### LINKED ISSUE #5196: Use real-time token balance in token balances API
### Problem

As a REST API user, I'd like to see my up to date token balance information when querying `/api/v1/tokens/{id}/balances`.

### Solution

* Change `/api/v1/tokens/{id}/balances` to query `token_account.balance` when no timestamp parameter is provided
* Update `docs/rest/README.md` to clarify which tables are actually used for all `token_balance` related endpoints.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
update token balances endpoint to use real time balance if no timestamp req param sent

**Related issue(s)**:

Fixes #5196 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/rest/README.md [hunk 1]
```diff
@@ -9,47 +9,49 @@ our [Swagger UI](https://mainnet-public.mirrornode.hedera.com/api/v1/docs/#/).
...
+| `/api/v1/tokens?account.id`                           | `entity`, `token`, `token_account`                                                                                                         |                                                               |
+| `/api/v1/tokens/:id`                                  | `custom_fee`, `entity`, `token`                                                                                                            |                                                               |
+| `/api/v1/tokens/:id/balances`                         | `token_balance`                                                                                                                            |                                                               |
+| `/api/v1/tokens/:id/balances?timestamp=`              | `token_account`, `record_file`                                                                                                             |                                                               |
```
Comment (steven-sheehy) [inline docs/rest/README.md:47]: The tables for these two rows should be swapped.

File: docs/rest/README.md [hunk 2]
```diff
@@ -9,47 +9,49 @@ our [Swagger UI](https://mainnet-public.mirrornode.hedera.com/api/v1/docs/#/).
...
+|-------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|
+| `/api/v1/accounts`                                    | `entity`, `token_balance`                                                                                                                  | Entity tables first used to filter, then joined w/ balances   |
+| `/api/v1/accounts?balance=false`                      | `entity`                                                                                                                                   | Balance tables skipped                                        |
+| `/api/v1/accounts/:idOrAlias`                         | `crypto_transfer`, `entity`, `record_file`, `token_balance`, `token_transfer`, `transaction`                                               | Transfers & transactions are present only for legacy reasons. |
```
Comment (steven-sheehy) [inline docs/rest/README.md:16]: We shouldn't use token_balance without a timestamp present. It should use token_account.

File: docs/rest/README.md [hunk 3]
```diff
@@ -9,47 +9,49 @@ our [Swagger UI](https://mainnet-public.mirrornode.hedera.com/api/v1/docs/#/).
...
+| `/api/v1/tokens/:id/balances?timestamp=`              | `token_account`, `record_file`                                                                                                             |                                                               |
+| `/api/v1/tokens/:id/balances?account.publickey`       | `entity`, `token_balance`                                                                                                                  |                                                               |
+| `/api/v1/tokens/:id/nfts`                             | `entity`, `nft`                                                                                                                            |                                                               |
+| `/api/v1/tokens/:id/nfts/:serial`                     | `entity`, `nft`                                                                                                                            |                                                               |
```
Comment (steven-sheehy) [inline docs/rest/README.md:50]: While you're in here, can you remove nft_transfer?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by edwin-greene — APPROVED** [review]
Looks good

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**